src/signer/canister/tests/it/canister/cycles_depositor.rs

```diff
@@ -54,7 +54,7 @@ impl PicCanisterTrait for CyclesDepositorPic {
 }
 
 impl CyclesDepositorPic {
-    pub fn deposit(&self, _caller: Principal, arg0: &DepositArg) -> Result<DepositResult, String> {
-        self.update_one(self.canister_id, "deposit", arg0)
+    pub fn deposit(&self, caller: Principal, arg0: &DepositArg) -> Result<DepositResult, String> {
+        self.update(caller, "deposit", (arg0,))
```

Comment:
The autogenerated bindings now all use `update` instead of `update_one`.  Oh, and the update_one bindings had a bug, using the wrong caller.  That is fixed now.  This doesn't change the two e2e tests that are currently enabled but it really matters when we get into the payment tests.